src/sarif_converter.ts

```diff
@@ -69,10 +69,11 @@ const getTools = (analysisResults: IAnalysisResult, suggestions: ISarifSuggestio
         level: severity,
       },
       help: {
+        markdown: suggestion.text,
         text: suggestion.message,
```

Comment:
we better remove this line. 
* one reason that [it](https://docs.oasis-open.org/sarif/sarif/v2.1.0/os/sarif-v2.1.0-os.html#_Toc34317473) should match the `help.markdown` content - just unformatted ..
* another reason is that `suggestion.message` may be instantiated with values of a specific suggestion instance, containing some symbol names from the code. `rule` should contain the common general data of a rule.

Reply:
suggestion.text != suggestion.message.

suggetion.text will contain only new data in markdown, or empty string.

Is it OK for us to return empty string in markdown here?

Reply:
> suggestion.text != suggestion.message.

In this comment I'm talking about `help.text`, not `help.markdown`.
`help.text` should be either empty or an unformatted version of `suggestion.text`, but it shouldn't be  `suggestion.message` which should only go to `result` (not to `rule`)

Reply:
done